tests/performance_sitespeed_io.py

```diff
@@ -210,6 +220,9 @@ def validate_on_mobile(url):
     if 'nt' in os.name:
         arg = '--shm-size=1g -b chrome --mobile true --connectivity.profile 3gfast --visualMetrics true --plugins.remove screenshot --speedIndex true --browsertime.videoParams.createFilmstrip false --browsertime.chrome.args ignore-certificate-errors -n {0} {1}'.format(
             get_config_or_default('SITESPEED_ITERATIONS'), url)
+    if 'docker' == WEBPERF_RUNNER:
+        arg = '--shm-size=1g -b chrome --mobile true --connectivity.profile 3gfast --visualMetrics false --plugins.remove screenshot --speedIndex false --browsertime.videoParams.createFilmstrip false --browsertime.chrome.args ignore-certificate-errors -n {0} {1}'.format(
```

Comment:
@krompaco why do we disabled speedIndex?

Reply:
Should be no change in this file now.